internal/pkg/armeria/command-handlers.go

```diff
@@ -134,6 +134,76 @@ func handleLookCommand(ctx *CommandContext) {
 	}
 }
 
+func handleGlanceCommand(ctx *CommandContext) {
+	r := ctx.Character.Room()
+
+	var objNames []string
+	for _, o := range r.Here().All() {
+		obj := o.(ContainerObject)
+
+		if obj.Type() == ContainerObjectTypeCharacter && obj.(*Character).Player() == nil {
+			continue
+		}
+
+		if obj.ID() != ctx.Character.ID() {
+			objNames = append(objNames, obj.FormattedName())
+		}
+	}
+
+	var withYou string
+	if len(objNames) > 0 {
+		withYou = fmt.Sprintf("\nHere with you: %s.", strings.Join(objNames, ", "))
+	}
+
+	ar := r.AdjacentRooms()
+	var validDirs []string
+	if ar.North != nil {
```

Comment:
I will probably break this out into its own function for use in `/look` and `/glance` to reduce the clutter here when showing valid directions.